pkg/machines/hostvmslist.jsx

```diff
@@ -463,41 +467,64 @@ Vm.propTypes = {
 /**
  * List of all VMs defined on this host
  */
-const HostVmsList = ({ vms, config, dispatch, actions }) => {
-    if (vms.length === 0) {
-        return (<div className='container-fluid'>
-            <NoVm />
-        </div>);
+class HostVmsList extends React.Component {
+    constructor(props) {
+        super(props);
+        this.deviceProxyHandler = this.deviceProxyHandler.bind(this);
+        this.state = { hostDevices: []};
```

Comment:
This looks  obsolete after your cleanup. Did you just forget to remove it, or am I missing something?

Reply:
My fault, thank you :)